src/bin/vip-slowlogs.ts

```diff
@@ -188,26 +191,30 @@ void command( {
 	envContext: true,
 	format: true,
 	module: 'slowlogs',
+	usage: baseUsage,
 } )
 	.option( 'limit', 'The maximum number of log lines', 500 )
 	.option( 'format', 'Output the log lines in CSV or JSON format', 'text' )
 	.examples( [
 		{
-			description: 'Get the most recent app slowlogs',
-			usage: 'vip @mysite.production slowlogs',
+			description:
+				'Retrieve up to 500 of the most recent entries from the MySQL slow query logs in the default text format.',
+			usage: exampleUsage,
 		},
 		{
-			usage: 'vip @mysite.production slowlogs --limit 100',
-			description: 'Get the most recent 100 slowlog entries',
+			description:
+				'Retrieve the 100 most recent entries from the MySQL slow query logs in the default text format.',
```

Comment:
I like your "up to 500" clarification in the default command setting, and I think we should continue with the "up to" clarification even when it's limited to 100. If a customer queries the slow query logs for an environment whose logs are are delightfully quiet, we don't want then to think there is something wrong if only 8 log entries are returned.